libs/utils/src/extract_cdf_ui_strings.test.ts

```diff
@@ -72,26 +72,198 @@ const tests: Record<ElectionStringKey, () => void> = {
   },
 
   [ElectionStringKey.CANDIDATE_NAME]() {
-    // TODO(kofi): Implement
+    const originalCandidates = assertDefined(ORIGINAL_ELECTION.Candidate);
+
+    const uiStrings = extractCdfUiStrings({
+      ...testCdfBallotDefinition,
+      Election: [
+        {
+          ...ORIGINAL_ELECTION,
+          Candidate: [
+            {
+              ...assertDefined(originalCandidates[0]),
+              '@id': 'candidate1',
+              BallotName: buildInternationalizedText({
+                [LanguageCode.ENGLISH]: 'Pinky and The Brain',
+                [LanguageCode.SPANISH]: 'Pinky y The Brain',
```

Comment:
😆

Reply:
Except it looks like it's ["Pinky y Cerebro"](https://www.youtube.com/watch?v=HKbPt5RcTXk) 🧠

Reply:
Omg amazing 😆 - must fix immediately

Also, hearing the theme song in Spanish just made my day